Sources/Quick/Example.swift

```diff
@@ -136,6 +119,65 @@ final public class Example: _ExampleBase {
         }
         return aggregateFlags
     }
+	
+	private func reportSkippedTest(_ testSkippedError: XCTSkip) {
+		#if _runtime(_Native)
+			return // This functionality is Only supported by Apple's proprietary XCTest, not by swift-corelibs-xctest
+		#elseif _runtime(_ObjC)
+			// Uses an internal type "XCTSkippedTestContext"
+			let skippedTestContext = testSkippedError.errorUserInfo["XCTestErrorUserInfoKeySkippedTestContext"]! as! NSObject
+		
+			let sourceCodeContext = skippedTestContext.value(forKey: "sourceCodeContext")! as! XCTSourceCodeContext
+			
+			guard let testRun = QuickSpec.current.testRun else { fatalError() }
+			
+			let selector = NSSelectorFromString("recordSkipWithDescription:sourceCodeContext:")
+		
+			guard testRun.responds(to: selector) else {
+				print("""
+					[Quick Warning]: The internals of Apple's XCTestCaseRun have changed, and it no longer responds to
+					the \(NSStringFromSelector(selector)) message necessary to report skipped tests to Xcode.
+					
+					If nobody else has done so yet, please submit an issue to https://github.com/Quick/Quick/issues
+					
+					For now, we'll just benignly ignore skipped tests.
+				""")
+				return
+			}
+		
+			testRun.perform(selector, with: testSkippedError.message, with: sourceCodeContext)
```

Comment:
Would it be better to add this in a header file, and make the call-site "normal" as far as Swift is concerned?

Reply:
I'm okay using `perform`.